src/mapGenerator/Defines.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2005 - 2015 Settlers Freaks (sf-team at siedler25.org)
+//
+// This file is part of Return To The Roots.
+//
+// Return To The Roots is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// Return To The Roots is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Return To The Roots. If not, see <http://www.gnu.org/licenses/>.
+
+#ifndef Defines_h__
+#define Defines_h__
+
+// http://settlers2.net/documentation/world-map-file-format-wldswd/
+
+// map types
+#define TYPE_GREENLAND                 0x00
+#define TYPE_WASTELAND                 0x01
+#define TYPE_WINTER                    0x02
+
+// texture types
+#define TEXTURE_STEPPE_MEADOW1         0x00
+#define TEXTURE_STEPPE_MEADOW1_HARBOUR 0x40
+#define TEXTURE_MINING1                0x01
+#define TEXTURE_MINING2                0x0B
+#define TEXTURE_MINING3                0x0C
+#define TEXTURE_MINING4                0x0D
+#define TEXTURE_MINING_MEADOW          0x12
+#define TEXTURE_MINING_MEADOW_HARBOUR  0x52
+#define TEXTURE_SNOW                   0x02
+#define TEXTURE_SWAMP                  0x03
+#define TEXTURE_STEPPE                 0x04
+#define TEXTURE_WATER                  0x05
+#define TEXTURE_WATER_                 0x06
+#define TEXTURE_STEPPE_                0x07
+#define TEXTURE_MEADOW1                0x08
+#define TEXTURE_MEADOW1_HARBOUR        0x48
+#define TEXTURE_MEADOW2                0x09
+#define TEXTURE_MEADOW2_HARBOUR        0x49
+#define TEXTURE_MEADOW3                0x0A
+#define TEXTURE_MEADOW3_HARBOUR        0x4A
+#define TEXTURE_STEPPE_MEADOW2         0x0E
+#define TEXTURE_STEPPE_MEADOW2_HARBOUR 0x4E
+#define TEXTURE_FLOWER                 0x0F
+#define TEXTURE_FLOWER_HARBOUR         0x4F
+#define TEXTURE_LAVA                   0x10
+#define TEXTURE_COLOR                  0x11
+#define TEXTURE_WATER__                0x13
+#define TEXTURE_STEPPE__               0x80
+#define TEXTURE_STEPPE___              0x84
```

Comment:
Yeah, I found the TerrainTypes now, too, so I'm going to use them :-) 

Must really look like i only copy pasted code together. Give me some time and I'll try to get more into the code base as it is. My vacation start at 9th of December, then I get lots of time to fix this. I've been more focused on getting nice maps generated, cause my brother has been pushing me since he knows this project. Finally I could show something hehe.